tree/tree/test/BulkApi.cxx

```diff
@@ -74,48 +82,108 @@ TEST_F(BulkApiTest, stdRead)
    delete hfile;
 }
 
-TEST_F(BulkApiTest, simpleRead)
+void SimpleReadFunc(const char *filename, const char *treename)
 {
-   auto hfile = TFile::Open(fFileName.c_str());
-   printf("Starting read of file %s.\n", fFileName.c_str());
+   auto hfile = TFile::Open(filename);
+   printf("Starting read of file %s.\n", filename);
```

Comment:
Should this and the following `printf`s become a `R__INFO` so that the verbosity of the test can be controlled?

Reply:
Probably, I just "continued" the style of the test ... i.e. let's do that in another PR.